modules/Expectation.js

```diff
@@ -294,72 +290,37 @@ class Expectation {
   }
 
   toInclude(value, compareValues, message) {
-    assert(
-      isArray(this.actual) || isObject(this.actual) || typeof this.actual === 'string',
-      'The "actual" argument in expect(actual).toInclude() must be an array, object, or a string'
-    )
-
     if (typeof compareValues === 'string') {
       message = compareValues
       compareValues = null
     }
 
-    message = message || 'Expected %s to include %s'
-
-    if (isArray(this.actual)) {
-      assert(
-        arrayContains(this.actual, value, compareValues),
-        message,
-        this.actual,
-        value
-      )
-    } else if (isObject(this.actual)) {
-      assert(
-        objectContains(this.actual, value, compareValues),
-        message,
-        this.actual,
-        value
-      )
-    } else {
-      assert(
-        stringContains(this.actual, value),
-        message,
-        this.actual,
-        value
-      )
-    }
+    containsHelper(
```

Comment:
I know this probably sounds silly, but I think I'd prefer to leave the branching in this method rather than tuck it away under `containsHelper`. It personally makes it easier for me to read the code and figure out e.g. what kinds of `actual` values `toInclude` accepts.